.circleci/config.yml

```diff
@@ -257,12 +257,27 @@ workflows:
   securedrop_ci:
     jobs:
       - lint
-      - trusty-app-tests
-      - xenial-app-tests
-      - admin-tests
-      - updater-gui-tests
+      - xenial-app-tests:
+          filters:
+            branches:
+              ignore:
+                - /docs-.*/
+      - admin-tests:
+          filters:
+            branches:
+              ignore:
+                - /docs-.*/
+      - updater-gui-tests:
+          filters:
+            branches:
+              ignore:
+                - /docs-.*/
       - static-analysis-and-no-known-cves
       - staging-test-with-rebase:
```

Comment:
Should we also run Xenial infra tests on each PR instead of the default (Trusty) tests?